code/game/objects/items/weapons/twohanded.dm

```diff
@@ -761,3 +763,66 @@
 				if(prob(4))
 					charged++
 					user.visible_message("<span class='notice'>The axe starts to emit an electric buzz!</span>")
+
+
+///Honkmother's Wrath///
+/obj/item/weapon/twohanded/chainsaw/honkmother
+	icon_state = "chainsaw0"
+	name = "Honkmother's Wrath"
+	desc = "The blade of the Honkmother herself. It hungers for blood, draining cowards who weild it, while rewarding thoes who kill in her name."
+	wieldsound = 'sound/weapons/chainsawstart.ogg'
+	armour_penetration = 60
+	attack_verb = list("SEEN", "LAUGHED", "PRANKED", "HONKED", "CUT ABOVE", "TIMBERED", "HATCHETED")
+
+/obj/item/weapon/twohanded/chainsaw/honkmother/update_icon()
+	if(wielded)
+		icon_state = "chainsaw[wielded]"
+	else
+		icon_state = "chainsaw0"
+
+/obj/item/weapon/twohanded/chainsaw/honkmother/attack(mob/target as mob, mob/living/user as mob)

```

Comment:
The icon state bits are placeholders until I can get  a sprite. They can be removed if needed but I think I just got a unique sprite.